package.json

```diff
@@ -14,16 +14,16 @@
     "design system",
     "pattern library"
   ],
+  "main": "packages/site/index.js",
   "workspaces": {
     "nohoist": [],
     "packages": [
       "docs",
       "packages/**"
     ]
   },
-  "main": "packages/site/index.js",
   "scripts": {
-    "build": "lerna run build",
+    "build": "lerna run build --ignore @pluralsight/ps-design-system-docs",
```

Comment:
why ignore docs on build?

Reply:
because it was slow. i'm actually going to change up the postinstall not to build. there will now be a build step you'll need to do after installation